src/EnergyPlus/ChillerElectricASHRAE205.cc

```diff
@@ -1443,13 +1429,22 @@ void ASHRAE205ChillerSpecs::calculate(EnergyPlusData &state, Real64 &MyLoad, boo
     } else if (this->ChillerPartLoadRatio < 1.0) // Modulating
     {
         // Use performance map to find the fractional sequence number (which most closely matches our part load)
-        auto f = std::bind(&ASHRAE205ChillerSpecs::findCapacityResidual, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
         Real64 constexpr accuracy{0.0001};
         int constexpr maxIter{500};
         int solFla{0};
-        std::array<Real64, 4> par{{MyLoad, RunFlag ? 1.0 : 0.0, 1.0}}; // Initialize iteration parameters for RegulaFalsi function
+        auto f = [&MyLoad, this](Real64 partLoadSeqNum) {
```

Comment:
Same comment, `=MyLoad`. Only pass by reference if you are trying to return a value or if you are passing something large like `state` or `this` and don't want a giant local copy.

Reply:
Another small note, `this` is just the instance pointer, so it is passed "by value" (or I guess actually by pointer).  In fact it's invalid to try to capture `&this`:

```
<source>:11:37: error: 'this' cannot be captured by reference
   11 |         auto lambda_with_capture = [&this](int x) {
      |                                     ^
```

Reply:
You know why that is, right? `this` is a language construct that means "the address of the current object", it's not an actual variable that you can take the address of. That's also (partly) why using `this` is optional in most situations. If `this` were a real variable, its use would be mandatory.